.changeset/unlucky-teachers-crash.md

```diff
@@ -0,0 +1,10 @@
+---
+"@wso2is/admin.webhooks.v1": minor
+"@wso2is/console": minor
+"@wso2is/i18n": minor
+"@wso2is/admin.actions.v1": patch
+"@wso2is/admin.core.v1": patch
+"@wso2is/features": patch
```

Comment:
Let's remove this, since we specify individual feature modules in the changeset.

```suggestion
```